portal-api/src/now.js

```diff
@@ -1,6 +1,4 @@
-const moment = require('moment');
-
 // one place that defines how we store timestamps..
 //  at least if we decide we don't like current implentation
 //  we just change it here + fix the results..
-module.exports = () => moment().utc().valueOf().toString();
+module.exports = () => new Date().valueOf().toString();
```

Comment:
Although it doesnt matter particularly it's more performant to write it as: 
```suggestion
module.exports = () => Date.now().toString();
```


Also -- I dislike this just being called `now`, i'd prefer a function called `getNowAsEpochMillisecondString` maybe? What do you think? Because 1. it's in ms and we have several areas where this is actually `seconds`, so we should be specific, and 2. string is surprising (as shown as we are adding `toString` on it). I know it's a slightly java-esq long naming strategry for the function so keen for your thoughts.